src/ariel-os-storage/src/lib.rs

```diff
@@ -131,6 +136,8 @@ where
 /// All items in flash have to be read and deserialized to find the items with the key.
 /// This is unlikely to be cached well.
 /// </div>
+// STM32 flash drivers do not implement `MultiwriteNorFlash`.
+#[cfg(not(context = "stm32"))]
```

Comment:
I'd prefer this to be like `cfg(has-MultiwriteNorFlash)`, let's revisit when the next comes around.